src/Core/src/Handlers/IndicatorView/IndicatorViewHandler.Windows.cs

```diff
@@ -5,27 +5,25 @@
 
 namespace Microsoft.Maui.Handlers
 {
-	public partial class IndicatorViewHandler : ViewHandler<IIndicatorView, ItemsControl>
+	public partial class IndicatorViewHandler : ViewHandler<IIndicatorView, MauiPageControl>
 	{
-		MauiPageControl? MauiPagerControl => PlatformView as MauiPageControl;
+		protected override MauiPageControl CreatePlatformView() => new MauiPageControl();
```

Comment:
This goes a bit beyond the scope of the interface-refactor, but exposing ` the change seemed to be beneficial for all 3 platforms. `MauiPageControl` is a public type and the casting felt counter-intuitive.

Reply:
👍

Reply:
@rmarinho any issues with this?

Reply:
I tried to similar stuff with `RadioButton`, trying to scope it's handler to `AppCompatRadioButton` instead of `View`, and faced an issue with Controls.Core/HandlerImpl/RadioButton which has a `PlatformViewFactory` override creating a view that doesn't derive from `AppCompatRadioButton`.

![image](https://user-images.githubusercontent.com/6835152/155228984-1271b9fb-4dcc-4d5f-b744-ec829bed7ecb.png)

![image](https://user-images.githubusercontent.com/6835152/155229048-d9a87495-df63-4516-9bbb-be66e2090bc0.png)

I wonder how far we can go with specialization? Is there some general rule?